package/MDAnalysis/analysis/align.py

```diff
@@ -332,6 +384,9 @@ def alignto(mobile, reference, select="all", mass_weighted=False,
        Uses :func:`get_matching_atoms` to work with incomplete selections
        and new *strict* keyword. The new default is to be lenient whereas
        the old behavior was the equivalent of *strict* = ``True``.
+
+    .. versionchanged:: 0.15.1
```

Comment:
I usually grep the repo for these and fix them on release. Not sure if
that's the best way though.

On Tue, 31 May 2016, 12:31 kain88-de, <notifications@github.com> wrote:

> In package/MDAnalysis/analysis/align.py
> <https://github.com/MDAnalysis/mdanalysis/pull/856#discussion_r65165053>:
>
> > @@ -332,6 +384,9 @@ def alignto(mobile, reference, select="all", mass_weighted=False,
> >         Uses :func:`get_matching_atoms` to work with incomplete selections
> >         and new *strict* keyword. The new default is to be lenient whereas
> >         the old behavior was the equivalent of *strict* = ``True``.
> > +
> > +    .. versionchanged:: 0.15.1
>
> @richardjgowers <https://github.com/richardjgowers> How should we handle
> things like this in the release when we decide to update the version number
> further then originally anticipated.
>
> —
> You are receiving this because you were mentioned.
>
>
> Reply to this email directly, view it on GitHub
> <https://github.com/MDAnalysis/mdanalysis/pull/856/files/3dd1dff2df1b585040e7b346b2886e08741bac8f#r65165053>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AI0jB-1Y0S4PoOvircN0vXN_U98bDNSAks5qHBwMgaJpZM4Ig_Re>
> .
>